packages/core/bootstrap/src/lib/middleware/burst-limit/index.ts

```diff
@@ -22,14 +22,21 @@ const availableSecondLimitCapacity = async (
   store: Store<BurstLimitState>,
   burstCapacity1s: number,
 ) => {
+  const enabled = parseBool(getEnv('RATE_LIMIT_CAPACITY'))
+  const defaultSecondslimit = 429
+
   for (let retry = SECOND_LIMIT_RETRIES; retry > 0; retry--) {
     store.dispatch(actions.updateIntervals())
     const state = store.getState()
     const { requests }: { requests: RequestsState } = state
 
     const observedRequestsInSecond = selectTotalNumberOfRequestsFor(requests, IntervalNames.SECOND)
 
-    if (observedRequestsInSecond > burstCapacity1s) {
+    if (enabled && observedRequestsInSecond >= defaultSecondslimit) {
```

Comment:
I don't see why we're checking against `defaultSecondslimit` here, can't we do this as part of the if check below?

Reply:
As far as I understand it, I have to validate that the RATE_LIMIT_CAPACITY is configured and that it reaches a certain number of requests per second, then putting it inside the conditional would alter the default conditions of the requests. Is this correct?